yb-voyager/cmd/importSchemaYugabyteDB.go

```diff
@@ -99,6 +99,14 @@ func YugabyteDBImportSchema(target *tgtdb.Target, exportDir string) {
 							os.Exit(1)
 						}
 					}
+				} else if strings.Contains(err.Error(), "multiple primary keys") {
```

Comment:
Shouldn't this issue be flagged by `analyze-schema` and corrected by the user. We can't just ignore this error.

Reply:
I believe this is PR is to help with the case of the user entering `--ignore-exist`. The DDL is sound as it introduces a PK constraint through ALTER TABLE, but this particular case is skipped with regards to the usage of this flag